examples/ads.amp.html

```diff
@@ -401,6 +402,13 @@ <h2 id="atomx">Atomx</h2>
       data-id="1234">
   </amp-ad>
 
+  <h2 id="bidtellect">Bidtellect</h2>
```

Comment:
only data-t and data-pid are required. the others are optional. You are not seeing an ad because I put two random numbers there. We nee this to start using google amp placement for one of our clients.